themes/classic/_dev/js/product.js

```diff
@@ -158,4 +159,18 @@ $(document).ready(() => {
       }
     });
   }
+
+  function addJsProductTabActiveSelector() {
+    const nav = $(prestashop.themeSelectors.product.tabs);
+    nav.on('show.bs.tab', (e) => {
+      const target = $(e.target);
+      target.addClass('js-product-nav-active');
+      $(target.attr('href')).addClass('js-product-tab-active');
```

Comment:
maybe a bit overkill but probably should go to the selectors map 🤷

Reply:
Franck get out of his body please !